compiler/rustc_hir_analysis/src/delegation.rs

```diff
@@ -1,11 +1,12 @@
 use std::assert_matches::debug_assert_matches;
```

Comment:
Could you add a top level comment here linking to the explainer?

Reply:
link to tracking issue was added instead as it is more comprehensive.